lib/RemoteAST/RemoteAST.cpp

```diff
@@ -126,11 +128,6 @@ class RemoteASTTypeBuilder {
 
   Demangle::NodeFactory &getNodeFactory() { return Factory; }
 
-  Type createBuiltinType(const std::string &mangledName) {
-    // TODO
-    return Type();
-  }
-
   NominalTypeDecl *createNominalTypeDecl(StringRef mangledName) {
```

Comment:
I don't understand why you're not moving the rest of this visitor to the new header too?